src/components/Adder.js

```diff
@@ -48,7 +51,7 @@ function Adder() {
           <option value="Economy">Economy</option>
           <option value="Action">Action</option>
         </select>
-        <button type="submit" onClick={submitBookToStore}>
+        <button className="btn-1" type="submit" onClick={submitBookToStore}>
           ADD BOOK
         </button>
       </div>
```

Comment:
- [ ] Kindly, add padding-bottom, the add book button and placeholder seem to cover on the bottom of the pahe

Reply:
:ok_man: Thanks for your time and for calling me to save my own time. It was a nice gesture from you. :+1: 
I had been implementing every change that you required and now the App is improved.
-  Added a bottom padding.
- Updated the Readme file, you did some mistakes in that file.
- Removed unnecessary files in your root folder to look cleaner.